server/src/main/java/org/kaazing/gateway/server/context/resolve/GatewayContextResolver.java

```diff
@@ -721,15 +722,17 @@ private void parseProperties(Node parent, ServiceProperties properties) {
         }
     }
 
-    private Collection<URI> resolveURIs(String[] acceptURIs) throws URISyntaxException {
-        Collection<URI> urisWithPort = new HashSet<>();
+    private Collection<String> resolveURIs(String[] acceptURIs) throws URISyntaxException {
+        Collection<String> urisWithPort = new HashSet<>();
         for (String uri : acceptURIs) {
-            URI resolvedURI = resolveURI(getCanonicalURI(uri, true));
+            // TODO: Add initial version of this to TcpResourceAddressSpi
+            String resolvedURI = /*resolveURI(*/(getCanonicalURI(uri, true)).toString();
```

Comment:
@jitsni Just readded resolveURI in GatewayContextResolver with my latest commit, thank you for your feedback. Initially I planned to add that resolution part to the ResourceAddressFactorySpi, as mentioned in the TODO:
// TODO: Add initial version of this to TcpResourceAddressSpi